.github/workflows/manual_corp_user_group_sync.yml

```
@@ -0,0 +1,31 @@
+name: Manually Sync Eras Corporate User Group Sync
+
+on:
+  workflow_dispatch:
+
+jobs:
+  manual_sync:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3.5.2
```

this should use a more up to date version, `actions/checkout` is up to 4.1.1 (listmonk-sync needs updating).
```suggestion
      uses: actions/checkout@v4.1.1
```